src/argon2.lua

```diff
@@ -63,6 +61,9 @@ local _M     = {
     _URL     = "https://github.com/thibaultCha/lua-argon2-ffi",
 }
 
+local encoded_len = function(salt_len, hash_len)
+    return 40 + (math.ceil(salt_len / 3) + math.ceil(hash_len / 3)) * 4
```

Comment:
According to [argon2 source](https://github.com/P-H-C/phc-winner-argon2/blob/master/src/encoding.c#L360), hashed password encoded like this: 
$`argon2_type`$v=`version integer`$m=`number`,t=`number`,p=`number`$`base64 encoded salt`$`base64 encoded hash`
For example $argon2i$v=19$m=65536,t=2,p=4$c29tZXNhbHQ$RdescudvJCsgt3ub+b+dWRWJTmaaJObG

In my formula, `40` is for `$argon2i$v=19$m=65536,t=2,p=4$` this part, I think 40 is enough for most cases. And the later part calculates the base64 encoded length for salt and hash.